po/fa.po

```diff
@@ -1881,12 +1953,21 @@ msgid ""
 "Hello, world!\n"
 "```"
 msgstr ""
+"```shell\n"
+"$ cd exercise\n"
+"$ cargo run\n"
+"   Compiling exercise v0.1.0 (/home/mgeisler/tmp/exercise)\n"
+"    Finished dev [unoptimized + debuginfo] target(s) in 0.75s\n"
+"     Running `target/debug/exercise`\n"
+"Hello, world!"
```

Comment:
This code block seems to not be closed:

```suggestion
"Hello, world!"
"```"
```

Reply:
Hey @DannyRavi, I cannot edit your branch, otherwise I would have applied this change myself. I checked locally that it makes `MDBOOK_BOOK__LANGUAGE=fa mdbook test` work without error.